calyx/src/passes/cell_share.rs

```diff
@@ -20,7 +19,7 @@ use std::collections::{HashMap, HashSet};
 /// This pass only renames uses of cells. [crate::passes::DeadCellRemoval] should be run after this
 /// to actually remove the definitions.
 pub struct CellShare {
-    live: LiveRangeAnalysis,
+    pub live: LiveRangeAnalysis,
```

Comment:
Why does this need to be `pub`?

Reply:
`sharing_components.rs` wanted to use it. Although I'm just realizing now that we can probably make `cell_share.rs` and `sharing_components.rs` the same file.

Reply:
That sounds like the right thing to do!